lib/react-dom.js

```diff
@@ -404,19 +404,281 @@ declare type $JSXIntrinsics = {
+  onPointerCancelCapture?: (SyntheticPointerEvent<E>) => void,
+  onPointerDown?: (SyntheticPointerEvent<E>) => void,
+  onPointerDownCapture?: (SyntheticPointerEvent<E>) => void,
+  onPointerEnter?: (SyntheticPointerEvent<E>) => void,
+  onPointerLeave?: (SyntheticPointerEvent<E>) => void,
+  onPointerMove?: (SyntheticPointerEvent<E>) => void,
+  onPointerMoveCapture?: (SyntheticPointerEvent<E>) => void,
+  onPointerOut?: (SyntheticPointerEvent<E>) => void,
+  onPointerOutCapture?: (SyntheticPointerEvent<E>) => void,
+  onPointerOver?: (SyntheticPointerEvent<E>) => void,
+  onPointerOverCapture?: (SyntheticPointerEvent<E>) => void,
+  onPointerUp?: (SyntheticPointerEvent<E>) => void,
+  onPointerUpCapture?: (SyntheticPointerEvent<E>) => void,
+  onProgress?: (SyntheticEvent<E>) => void,
+  onProgressCapture?: (SyntheticEvent<E>) => void,
+  onRateChange?: (SyntheticEvent<E>) => void,
+  onRateChangeCapture?: (SyntheticEvent<E>) => void,
+  onReset?: (SyntheticEvent<E>) => void,
+  onResetCapture?: (SyntheticEvent<E>) => void,
+  onScroll?: (SyntheticUIEvent<E>) => void,
+  onScrollCapture?: (SyntheticUIEvent<E>) => void,
+  onSeeked?: (SyntheticEvent<E>) => void,
+  onSeekedCapture?: (SyntheticEvent<E>) => void,
+  onSeeking?: (SyntheticEvent<E>) => void,
+  onSeekingCapture?: (SyntheticEvent<E>) => void,
+  onSelect?: (SyntheticEvent<E>) => void,
+  onSelectCapture?: (SyntheticEvent<E>) => void,
+  onStalled?: (SyntheticEvent<E>) => void,
+  onStalledCapture?: (SyntheticEvent<E>) => void,
+  onSubmit?: (SyntheticEvent<E>) => void,
+  onSubmitCapture?: (SyntheticEvent<E>) => void,
+  onSuspend?: (SyntheticEvent<E>) => void,
+  onSuspendCapture?: (SyntheticEvent<E>) => void,
+  onTimeUpdate?: (SyntheticEvent<E>) => void,
+  onTimeUpdateCapture?: (SyntheticEvent<E>) => void,
+  onToggle?: (SyntheticEvent<E>) => void,
+  onToggleCapture?: (SyntheticEvent<E>) => void,
+  onTouchCancel?: (SyntheticTouchEvent<E>) => void,
+  onTouchCancelCapture?: (SyntheticTouchEvent<E>) => void,
+  onTouchEnd?: (SyntheticTouchEvent<E>) => void,
+  onTouchEndCapture?: (SyntheticTouchEvent<E>) => void,
+  onTouchMove?: (SyntheticEvent<E>) => void,
+  onTouchMoveCapture?: (SyntheticEvent<E>) => void,
+  onTouchStart?: (SyntheticTouchEvent<E>) => void,
+  onTouchStartCapture?: (SyntheticTouchEvent<E>) => void,
+  onTouchTap?: (SyntheticUIEvent<E>) => void,
+  onTouchTapCapture?: (SyntheticUIEvent<E>) => void,
+  onTransitionEnd?: (SyntheticTransitionEvent<E>) => void,
+  onTransitionEndCapture?: (SyntheticTransitionEvent<E>) => void,
+  onVolumeChange?: (SyntheticEvent<E>) => void,
+  onVolumeChangeCapture?: (SyntheticEvent<E>) => void,
+  onWaiting?: (SyntheticEvent<E>) => void,
+  onWaitingCapture?: (SyntheticEvent<E>) => void,
+  onWheel?: (SyntheticWheelEvent<E>) => void,
+  onWheelCapture?: (SyntheticWheelEvent<E>) => void,
+};
+
+type ReactDOM$ElementProps = {
+  children?: React$Node,
+  className?: string,
```

Comment:
This often appears to be `null`, `void` or `String`.

Reply:
true. But is `null` correct? Though, the defs can can always be made stricter later if decided so.

Reply:
made null possible.